### PR TITLE
Support for parallel evaluation in LightCurveEstimator

### PR DESCRIPTION
**Pull request by QRemy**
Support for parallel evaluation in LightCurveEstimator

### CODE REVIEW

**Review by bkhelifi — APPROVED** [review]
Thanks @QRemy. This is a feature that anyone will enjoy. I have nothing to add

File: gammapy/estimators/points/lightcurve.py [hunk 1]
```diff
@@ -75,11 +79,21 @@ class LightCurveEstimator(FluxPointsEstimator):
...
     tag = "LightCurveEstimator"
 
-    def __init__(self, time_intervals=None, atol="1e-6 s", **kwargs):
+    def __init__(self, time_intervals=None, atol="1e-6 s", n_jobs=1, **kwargs):
         self.time_intervals = time_intervals
         self.atol = u.Quantity(atol)
+        self.n_jobs = n_jobs
         super().__init__(**kwargs)
 
+    def _estimate_time_bin_flux(self, datasets_to_fit, dataset_names):
```
Comment (registerrier) [inline gammapy/estimators/points/lightcurve.py:88]: Why not adding this code directly into the `estimate_time_bin_flux` method?

**Review by registerrier — DISMISSED** [review]
Thanks @QRemy. See small inline comment.

**Review by registerrier — APPROVED** [review]
Thanks @QRemy. No more comments from my side.

**Pull request merged**